spec/requests/follows_update_spec.rb

```diff
@@ -12,7 +12,8 @@
   describe "PUT follows/:id" do
     it "updates follow points" do
       user.follow(tag)
-      put "/follows/#{Follow.last.id}", params: { follow: { points: 3.0 } }
+      put "/follows/#{Follow.last.id}", params: { follow: { explicit_points: 3.0 } }
```

Comment:
**nitpick (non-blocking):** I think it wouldn't hurt if we also use path helpers in our specs.

```suggestion
      put follows_path(Follow.last.id), params: { follow: { explicit_points: 3.0 } }
```

Reply:
✅